README.md

```diff
@@ -93,6 +93,18 @@ order by
 +---------------+----------------+------------------+--------+---------------+
 ```
 
+## Detections as Code with Powerpipe
+
+Pre-built dashboards and detections for the Nginx plugin are available in [Powerpipe](https://powerpipe.io) mods, helping you monitor and analyze activity across your Nginx servers.
+
+For example, the [Nginx Access Log Detections mod](https://hub.powerpipe.io/mods/turbot/tailpipe-mod-nginx-access-log-detections) scans your Nginx access logs for anomalies, such as sql injestion attacks on your web application.
+
+Dashboards and detections are [open source](https://github.com/topics/tailpipe-mod), allowing easy customization and collaboration.
+
+To get started, choose a mod from the [Powerpipe Hub](https://hub.powerpipe.io/?engines=tailpipe&q=nginx).
+
+![image](docs/images/nginx_access_log_detection_dashboard.png)
```

Comment:
Can we take an OWASP screenshot as well and use that here (we can keep the other screenshot you have as well and use it in docs/index.md)?